README.md

```diff
@@ -54,9 +54,11 @@ pub mod uppercase;
 In the markdown file, the unit test can be called using:
 
 ```markdown
-@[Test uppercase]({"stubs":["uppercase.rs"], "command":"cargo test"})
+@[Test uppercase]({"stubs":["uppercase.rs"], "buildCommand": "cargo build", "command":"cargo test string_tests::test_to_upper"})
 ```
 
+We specify a `buildCommand` to build the code in advance. The `command` only runs the `string_tests::test_to_upper` test here: this allows us to use the same code throughout the playground and illustrate different aspects of the code with a specific test for each run.
```

Comment:
I'm sorry, I was not too precise… The `buildCommand` is on the `techio.yml` file. It is called once when you build your image.